src/GHCMod.hs

```diff
@@ -277,20 +277,20 @@ File map docs:
 CLI options:
 * `--map-file "file1.hs=file2.hs"` can be used to tell
     ghc-mod that it should take source code for `file1.hs` from `file2.hs`.
-    `file1.hs` can be either full path, or path relative to project root.
+    `file1.hs` can be either full path, or path relative to current directory.
```

Comment:
The "project root" is whatever cradleRootDir is, no matter if it's a cabal project or not. In runGhcModT I always set cwd to cradleRootDir so all relative paths will be relative to the project root.